.circleci/config.yml

```diff
@@ -124,9 +124,9 @@ jobs:
           name: Get Ready
           command: |
             eval $PRE_TEST_COMMANDS;
-            ./meteor --help
-          # shouldn't take longer than 5 minutes
-          no_output_timeout: 5m
+            ./meteor --get-ready
```

Comment:
Not only did this solve our testing problems (🤞), but it demonstrates another strength of @abernix's Circle 2.0 configuration: this command runs once, before any test jobs, so it only has one chance to fail. If it fails, it fails fast, and we can easily re-run the workflow.